packages/debugger/src/panels/kernelSources/body.tsx

```diff
@@ -21,19 +23,149 @@ import { UseSignal } from '@jupyterlab/ui-components';
 import { IRenderMime } from '@jupyterlab/rendermime';
 
 /**
- * The class name added to the filterbox node.
+ * CSS class names.
  */
 const FILTERBOX_CLASS = 'jp-DebuggerKernelSource-filterBox';
+const FILTERBOX_HIDDEN_CLASS = 'jp-DebuggerKernelSource-filterBox-hidden';
+const SOURCE_CLASS = 'jp-DebuggerKernelSource-source';
+const DIR_CLASS = 'jp-DebuggerKernelSource-dir';
 
 /**
- * The class name added to hide the filterbox node.
+ * Props for a tree node.
  */
-const FILTERBOX_HIDDEN_CLASS = 'jp-DebuggerKernelSource-filterBox-hidden';
+interface ITreeNodeProps {
+  name: string;
+  path: string;
+  modName: string;
```

Comment:
I personally always prefer long explicit names over shortcuts:

```suggestion
  moduleName: string;
```